faust-state/src/lib.rs

```diff
@@ -270,6 +270,39 @@ impl Node {
             0.
         }
     }
+
+    /// The minimum value for this parameter.
+    ///
+    /// This value is an indication of the DSP and is not enforced.
+    pub fn min(&self) -> f32 {
```

Comment:
Maybe better to return an Option instead of 0.? Otherwise it's not clear if the min value really is 0. or there is none set?

Reply:
That's likely better, I'll have to dive a bit deeper in there, as I don't understand in which condition the props would be None here

Reply:
Done! It's for Button that there is no "Props" object associated

Reply:
I've opened an alternative PR #12 that keeps a richer set of data so there is no more Option<Props> that depends on the kind of parameter.